extensions/reactive-db2-client/runtime/src/main/java/io/quarkus/reactive/db2/client/runtime/DB2PoolRecorder.java

```diff
@@ -50,6 +52,33 @@ public class DB2PoolRecorder {
     private static final TypeLiteral<Instance<DB2PoolCreator>> POOL_CREATOR_TYPE_LITERAL = new TypeLiteral<>() {
     };
 
+    private final RuntimeValue<DataSourcesRuntimeConfig> runtimeConfig;
+    private final RuntimeValue<DataSourcesReactiveRuntimeConfig> reactiveRuntimeConfig;
```

Comment:
IIUC a new recorder instance is created for each build step invocation?

Reply:
I'm not sure, but at static/runtime init I'd expect one recorder instance per build step, not one per call made from a build step.

Why this question though?